docs/resources/files.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Storage"
+---
+# databricks_files Resource
+
+This is a resource that lets you upload and download files upto 2GB in octet-stream.
+
+## Example Usage
+
+In order to manage a file on Databricks File System with Terraform, you must specify the `source` attribute containing the full path to the file on the local filesystem.
+
+```hcl
+resource "databricks_files" "this" {
+  source = "${path.module}/main.tf"
+  path   = "/tmp/main.tf"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `source` - The full absolute path to the file. Conflicts with `content_base64`.
```

Comment:
we don't have `content_base64` in this resource, but it would be really useful to have it similar to `databricks_dbfs_file`, especially for small things, like, init scripts

Reply:
Added